common/tvm_utility/CMakeLists.txt

```diff
@@ -67,10 +46,11 @@ if(BUILD_TESTING)
     # the folder name becomes the test case name
     file(RELATIVE_PATH TEST_CASE_NAME ${CMAKE_CURRENT_LIST_DIR}/test ${TEST_FOLDER})
 
-    # Test if files exist. The result is set in ${TEST_CASE_NAME}_FOUND
-    autoware_check_neural_network(${TEST_CASE_NAME} "${${PROJECT_NAME}_BACKEND}")
+    # Get neural network.
+    set(NN_DEPENDENCY "")
+    get_neural_network(${TEST_CASE_NAME} ${${PROJECT_NAME}_BACKEND} NN_DEPENDENCY)
```

Comment:
It seems that `tvm_utility` is now downloading required dependencies. If we add a new package such as `lidar_centerpoint_tvm`, should we add a similar line of code like this in the `CMakeLists`? I feel like that the added package itself should handle such tasks because at this point we have no idea where networks we need.

Reply:
Please ignore the above comment